CRM/Case/XMLProcessor/Process.php

```diff
@@ -740,8 +732,7 @@ public function getMaxInstance($caseType, $activityTypeName = NULL) {
     $xml = $this->retrieve($caseType);
 
     if ($xml === FALSE) {
-      CRM_Core_Error::fatal();
-      return FALSE;
+      throw new CRM_Core_Exception('Unable to read supplied XML File');
```

Comment:
I might change the wording here, something like "Unable to locate xml definition for case type $caseType".

But it's minor since the typical way you'd get here is if you're on manage case and try to add a new activity, but if you're on manage case and the case type xml doesn't exist it would have had a fit already.

Reply:
The docblock here too.

And what? `@param null $activityTypeName`